### PR TITLE
Feature/update to polkadot v0 9 11

### PR DESCRIPTION
**Pull request by MRamanenkau**

### CODE REVIEW

File: bin/node/runtime/src/lib.rs [hunk 1]
```diff
@@ -405,6 +418,7 @@ impl pallet_balances::Config for Runtime {
 
 parameter_types! {
 	pub const TransactionByteFee: Balance = 10 * MILLICENTS;
+	pub const OperationalFeeMultiplier: u8 = 5;
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:421]: This variable introduced in scope of [transaction priority calculation rework](https://github.com/paritytech/substrate/issues/9317). It regulates priority of operational dispatches (which provide network capabilities).
I guess we can rely he on substrate team leave it as it is.

File: bin/node/runtime/src/lib.rs [hunk 2]
```diff
@@ -492,6 +507,11 @@ parameter_types! {
 }
 
 use frame_election_provider_support::onchain;
+impl onchain::Config for Runtime {
+	type Accuracy = Perbill;
+	type DataProvider = Staking;
+}
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:513]: Refactoring. [No logic change](https://github.com/paritytech/substrate/pull/9648)

File: bin/node/runtime/src/lib.rs [hunk 3]
```diff
@@ -887,6 +939,10 @@ parameter_types! {
 	pub const ImOnlineUnsignedPriority: TransactionPriority = TransactionPriority::max_value();
 	/// We prioritize im-online heartbeats over election solution submission.
 	pub const StakingUnsignedPriority: TransactionPriority = TransactionPriority::max_value() / 2;
+	pub const MaxAuthorities: u32 = 100;
+	pub const MaxKeys: u32 = 10_000;
+	pub const MaxPeerInHeartbeats: u32 = 10_000;
+	pub const MaxPeerDataEncodingSize: u32 = 1_000;
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:945]: [Refactoring](https://github.com/paritytech/substrate/pull/9654) of [pallet_im_online](https://docs.rs/pallet-im-online/latest/pallet_im_online/). Looks like these values are not very important and we can keep it

File: bin/node/runtime/src/lib.rs [hunk 4]
```diff
@@ -1066,6 +1128,9 @@ impl pallet_vesting::Config for Runtime {
 	type BlockNumberToBalance = ConvertInto;
 	type MinVestedTransfer = MinVestedTransfer;
 	type WeightInfo = pallet_vesting::weights::SubstrateWeight<Runtime>;
+	// `VestingInfo` encode length is 36bytes. 28 schedules gets encoded as 1009 bytes, which is the
+	// highest number of schedules that encodes less than 2^10.
+	const MAX_VESTING_SCHEDULES: u32 = 28;
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:1133]: [Support multiple vesting schedules per account. ](https://github.com/paritytech/substrate/pull/9202)
[The issue it solves](https://github.com/paritytech/substrate/issues/7101)

File: Cargo.lock [hunk 5]
```diff
@@ -2642,6 +2642,17 @@ dependencies = [
  "digest 0.9.0",
 ]
 
+[[package]]
+name = "hmac-drbg"
```
Comment (AndreiNavoichyk) [inline Cargo.lock:2646]: Should it be removed?

File: bin/node/runtime/Cargo.toml [hunk 6]
```diff
@@ -70,7 +75,9 @@ pallet-offences-benchmarking = { version = "4.0.0-dev", path = "../../../frame/o
 pallet-proxy = { version = "4.0.0-dev", default-features = false, path = "../../../frame/proxy" }
 pallet-randomness-collective-flip = { version = "4.0.0-dev", default-features = false, path = "../../../frame/randomness-collective-flip" }
 pallet-recovery = { version = "4.0.0-dev", default-features = false, path = "../../../frame/recovery" }
-pallet-session = { version = "4.0.0-dev", features = ["historical"], path = "../../../frame/session", default-features = false }
+pallet-session = { version = "4.0.0-dev", features = [
+	"historical",
```
Comment (AndreiNavoichyk) [inline bin/node/runtime/Cargo.toml:79]: Should this formatting be fixed?

File: bin/node/runtime/Cargo.toml [hunk 7]
```diff
@@ -50,6 +54,7 @@ frame-try-runtime = { version = "0.10.0-dev", default-features = false, path = "
 pallet-authority-discovery = { version = "4.0.0-dev", default-features = false, path = "../../../frame/authority-discovery" }
 pallet-authorship = { version = "4.0.0-dev", default-features = false, path = "../../../frame/authorship" }
 pallet-babe = { version = "4.0.0-dev", default-features = false, path = "../../../frame/babe" }
+pallet-bags-list = { version = "4.0.0-dev", default-features = false, path = "../../../frame/bags-list" }
```
Comment (AndreiNavoichyk) [inline bin/node/runtime/Cargo.toml:57]: Why do we need this?

Comment (MRamanenkau) [reply bin/node/runtime/Cargo.toml:57]: Check details [here](https://www.notion.so/cere/Testing-logs-1f7d5107060a432398bad8b6a15225f0#55e0d39b91ef457783aef353f6afba97)

File: Cargo.lock (hunk 5 above)
Comment (MRamanenkau) [reply Cargo.lock:2646]: Why? There are [packages](https://github.com/Cerebellum-Network/pos-network-node/blob/df9ecb657bccd0ff2229cdeaf6d0a6d6a3524bc6/Cargo.lock#L3810) which depend on it.

File: bin/node/runtime/Cargo.toml (hunk 6 above)
Comment (MRamanenkau) [reply bin/node/runtime/Cargo.toml:79]: No, it's Substrate's code

**Review by AndreiNavoichyk — DISMISSED** [review]

**Pull request merged**